server/server.go

```diff
@@ -78,9 +79,11 @@ type Server struct {
 // NewServer creates the pd server with given configuration.
 func NewServer(cfg *Config) (*Server, error) {
 	cfg.adjust()
+	log.Infof("PD config - %v", cfg)
```

Comment:
In main.go, we have already had a same log, like `log.Infof("PD init config - %v", cfg)`, so should we remove that one?